diff --git a/lisp/emacs-lisp/comp.el b/lisp/emacs-lisp/comp.el
index 8441b22..a4b8602 100644
--- a/lisp/emacs-lisp/comp.el
+++ b/lisp/emacs-lisp/comp.el
@@ -99,8 +99,10 @@ and above."
   :version "28.1")
 
 (defcustom native-comp-driver-options
+  (append
   (cond ((eq system-type 'darwin) '("-Wl,-w"))
         ((eq system-type 'cygwin) '("-Wl,-dynamicbase")))
+    '(@backendPath@))
   "Options passed verbatim to the native compiler's back-end driver.
 Note that not all options are meaningful; typically only the options
 affecting the assembler and linker are likely to be useful.
