diff --git Makefile.in Makefile.in
index 0b4df73..fc0c51f 100644
--- Makefile.in
+++ Makefile.in
@@ -973,7 +973,7 @@ test tests check:	force $(Program) $(TESTS_SUPPORT)
 	@-test -d tests || mkdir tests
 	@cp $(TESTS_SUPPORT) tests
 	@( cd $(srcdir)/tests && \
-		BUILD_DIR=$(BUILD_DIR) PATH=$(BUILD_DIR)/tests:$$PATH THIS_SH=$(THIS_SH) $(SHELL) ${TESTSCRIPT} )
+		BUILD_DIR=$(BUILD_DIR) PATH=$(BUILD_DIR)/tests:$$PATH THIS_SH=$(THIS_SH) $(SHELL) -e ${TESTSCRIPT} )
 
 symlinks:
 	$(SHELL) $(SUPPORT_SRC)fixlinks -s $(srcdir)
diff --git tests/run-alias tests/run-alias
index 3b4f4dc..a1db365 100644
--- tests/run-alias
+++ tests/run-alias
@@ -1,2 +1,3 @@
 ${THIS_SH} ./alias.tests > ${BASH_TSTOUT} 2>&1
 diff ${BASH_TSTOUT} alias.right && rm -f ${BASH_TSTOUT}
+#diff ${BASH_TSTOUT} alias.right || exit 1
diff --git tests/run-all tests/run-all
index 1f74923..69670a1 100644
--- tests/run-all
+++ tests/run-all
@@ -57,7 +57,7 @@ do
 	case $x in
 	$0|run-minimal|run-gprof)	;;
 	*.orig|*~) ;;
-	*)	echo $x ; sh $x ; rm -f ${BASH_TSTOUT} ;;
+	*)	echo $x ; sh -e $x ; rm -f ${BASH_TSTOUT} ;;
 	esac
 done
 
